trackma/ui/cli.py

```diff
@@ -990,19 +990,20 @@ def _make_list_json(self, showlist):
                 "title": title_str,
                 "current_episode": episode_str_current,
                 "total_episodes": episode_str_last,
-                "score": score
+                "score": score,
+                "airing_status": False,
+                "behind_status": False
             }
 
-            # Color title according to status
+            # Check if show is airing and if user is behind
             if show['status'] == utils.Status.AIRING:
                 estimate = utils.estimate_aired_episodes(show)
                 if estimate and show['my_progress'] < estimate:
                     # User is behind the (estimated) aired episode
-                    j["color"] = _COLOR_BEHIND
+                    j["airing_status"] = True;
```

Comment:
Actually, why not just dump `show['status']` here?

Reply:
Well, if I do that I get:
`TypeError: Object of type Status is not JSON serializable`
I also tried to simply have a variable named status and no if else statement. Same issue.
But if I simply print status it just says "Ongoing".

Perhaps I just didn't use it right?

Reply:
Yeah, you'll need to use `show['status'].value`.